test/extended/util/test.go

```diff
@@ -407,16 +407,6 @@ var (
 			`should idle the service and DeploymentConfig properly`, // idling with a single service and DeploymentConfig [Conformance]
 
 			`\[Feature:Volumes\]`, // storage team to investigate it post-rebase
-
-			// TODO: the following list of tests is disabled temporarily due to the fact
-			// that we're running kubelet 1.11 and these require 1.12. We will remove them
-			// post-rebase
-			`\[Feature:NodeAuthenticator\]`,
-			`PreemptionExecutionPath`,
```

Comment:
@ravisantoshgudimetla does this test should pass I'm seeing failures in https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/21904/pull-ci-openshift-origin-master-e2e-aws/3146 ?

Reply:
@sjenning do you know ^ the nodeauthenticator also fails for this PR every time.